tests/phpunit/CRM/RcBase/ConfigHeadlessTest.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+use CRM_RcBase_ExtensionUtil as E;
+use Civi\Test;
+use Civi\Test\HeadlessInterface;
+use Civi\Test\HookInterface;
+use Civi\Test\TransactionalInterface;
+
+const DEFAULT_CONFIGURATION = [
+    "Key1" => "value1",
+    "Key2" => 12,
+    "Key3" => true,
+    "Key4" => [],
+    "Key5" => [ "SubKey" => "Great success!", ],
+];
+const CONFIG_NAME = "rcBase_test";
+
+class TestConfig extends CRM_RcBase_Config {
+    public function defaultConfiguration(): array {
+        return DEFAULT_CONFIGURATION;
+    }
+}
+/**
+ * Config class test cases.
+ * It tests the testConfig class that extends from the config and implements the 
+ * defaultConfiguration method.
+ *
+ * @group headless
+ */
+class CRM_RcBase_ConfigHeadlessTest extends \PHPUnit\Framework\TestCase implements HeadlessInterface {
+
+
+    public function setUpHeadless() {
+        return Test::headless()
+            ->installMe(__DIR__)
+            ->apply();
+    }
+    /**
+     * Create a clean DB before running tests
+     *
+     * @throws CRM_Extension_Exception_ParseException
+     */
+    public static function setUpBeforeClass(): void {
+        Test::headless()
+            ->installMe(__DIR__)
+            ->apply(true);
+    }
+    /**
+     * Create a clean DB before running tests
+     *
+     * @throws CRM_Extension_Exception_ParseException
+     */
+    public static function tearDownAfterClass(): void {
```

Comment:
Why is this neat clean-up process?
Next running tests should setup and reset the DB for themselves anyway.

Reply:
When i have a before class with an install process, i think i should have an after class with the uninstall step. But it could be skipped.